src/lycophron/app.py

```diff
@@ -64,18 +63,17 @@ def recreate_project(self):
     def validate_project(self):
         if not self.project.is_project_initialized():
             raise ValueError("Project is not initialised!")
-        self.project.validate_project()
+        self.project.validate_project(self.config)
 
     def is_project_initialized(self):
         return self.project and self.project.is_project_initialized()
 
     def load_file(self, filename):
-        self.project.load_file(filename)
+        self.project.load_file(filename, self.config)
 
     def publish_records(self, num_records=None):
-        self.project.publish_records(
-            self.config["ZENODO_URL"], self.config["TOKEN"], num_records
-        )
+        publish_url = self.config["ZENODO_URL"] + "/api/deposit/depositions"
```

Comment:
Minor: this is likely to be changed by https://github.com/zenodo/rdm-project/issues/646 , since the URL is just used to initialize the `inveniordm-py` client